evm/rlp/headers.py

```diff
@@ -196,15 +207,57 @@ def __repr__(self):
 
     @property
     def hash(self):
-        return keccak(rlp.encode(self))
+        int_to_bytes32 = compose(
```

Comment:
I'd be good with this living in `evm.utils.numeric`.  I suspect there are other places in the code where we'd use it.

Reply:
I have moved it to `evm.utils.numeric`, and added two tests for it.
Hope that would help.